testing/integration/agent_long_test_apache.json

```diff
@@ -0,0 +1,252 @@
+{
+    "id": "5cca0416-2c8e-43bb-a12f-108088a2d19c",
+    "version": "WzY3NiwxXQ==",
+    "name": "apache-1",
+    "namespace": "",
+    "description": "",
+    "package": {
+      "name": "apache",
+      "title": "Apache HTTP Server",
+      "version": "1.17.0"
+    },
+    "enabled": true,
+    "policy_id": "cf3042c7-9086-463f-85c8-927bd8ec655b",
```

Comment:
This changes on every install does it not?

Reply:
We overwrite it with a UUID created at runtime, that's just a a placeholder.

Reply:
We can just remove it for the sake of clarity.